code/game/objects/structures/crates_lockers/closets/utility_closets.dm

```diff
@@ -41,11 +42,17 @@
 		if ("tank")
 			new /obj/item/tank/internals/oxygen(src)
 			new /obj/item/clothing/mask/breath(src)
+			new /obj/item/inflatable/shelter(src)
 
 		if ("both")
 			new /obj/item/tank/internals/emergency_oxygen(src)
 			new /obj/item/clothing/mask/breath(src)
 
+		if ("inflatables")
+			new /obj/item/tank/internals/emergency_oxygen(src)
+			new /obj/item/clothing/mask/breath(src)
+			new /obj/item/storage/briefcase/inflatable(src)
```

Comment:
Giving these out as crew standard emergency equipment might be a bit much?

Reply:
The inflatable case has a much lower weight then any of the other equipment sets. In playtesting it seemed like it was actually hard to find more then one or two cases on the station, so personally I think this is fine balance-wise.